scripts/jenkins/Jenkinsfile

```diff
@@ -46,20 +46,20 @@ def buildClosures(arg) {
                 stage("build-${platform}") {
                     rebuild = detectChanges()
                     docker.build("${platform}-master-test","${rebuild} -f ./scripts/docker/build-${platform}/Dockerfile.deps ./scripts/docker/build-${platform}").inside {
-                        checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'CleanBeforeCheckout'],[$class: 'RelativeTargetDirectory', relativeTargetDir: "${platform}/build"]], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/freeradius/freeradius-server']]])
+                        checkout([$class: 'GitSCM', branches: scm.branches, userRemoteConfigs: scm.userRemoteConfigs, extensions: [[$class: 'CleanBeforeCheckout'],[$class: 'RelativeTargetDirectory', relativeTargetDir: "${platform}/build"],[$class: 'CloneOption', depth: 0, noTags: false, reference: '', shallow: false]], submoduleCfg: []])
                         sh "cat /etc/os-release || cat /etc/redhat-release"
+                        def commit_num = readFile("./build-number").trim()
                         dir("${platform}/build") {
                             if (platform.contains("centos")) {
-                                sh 'export COMMIT_NUM=$(git describe --tags --long --match "release_*" --match "branch_*" | sed -e \'s/^.*-\\([0-9]*\\)-g[0-9a-f]*/\\1/\')  ; sed -i -e "s/^Release: .*$/Release: $COMMIT_NUM/" redhat/freeradius.spec'
-                                sh 'QA_RPATHS=0x0003 RADIUSD_VERSION_STRING=$(cat VERSION) make rpm'
+                                sh "sed -i -e \"s/^Release:.*\$/Release: ${commit_num}/\" redhat/freeradius.spec"
                                 sh 'RADIUSD_VERSION_STRING=$(cat VERSION) make rpm'
                                 sh "mv rpmbuild/RPMS/x86_64/*.rpm .."
                             } else {
                                 sh "apt-get install -y unixodbc-dev"
-                                sh label: '', script: '''version=$(dpkg-parsechangelog | grep "^Version: " | awk \'{print $2}\')
-                                commit_num=$(git describe --tags --long --match "release_*"  --match "branch_*" | sed -e \'s/^.*-\\([0-9]*\\)-g[0-9a-f]*/\\1/\')
-                                commit_msg="$(git log --oneline -1 $GIT_COMMIT)"
-                                dch -b -v ${version}-${commit_num} "$commit_msg"'''
+                                def version = sh (script: "dpkg-parsechangelog | grep '^Version: ' | awk '{print \$2}'", returnStdout: true).trim()
```

Comment:
Maybe: `awk '/Version/ { print $2}'`

Reply:
Yeah, 6 of one, a half-dozen of the other...